integration-test/pom.xml

```diff
@@ -160,4 +160,83 @@ println(new File(project.build.directory, "fake-hosts.txt").text)
+                --add-opens=java.base/java.nio=ALL-UNNAMED
+                --add-opens=java.base/java.net=ALL-UNNAMED
+                --add-opens=java.base/java.util=ALL-UNNAMED
+                --add-opens=java.base/java.util.concurrent=ALL-UNNAMED
+                --add-opens=java.base/java.util.concurrent.locks=ALL-UNNAMED
+                --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED
+                --add-opens=java.base/java.lang=ALL-UNNAMED
+                --add-opens=java.base/java.lang.invoke=ALL-UNNAMED
+                --add-opens=java.base/java.math=ALL-UNNAMED
+                --add-opens=java.sql/java.sql=ALL-UNNAMED
+                --add-opens=java.base/java.lang.reflect=ALL-UNNAMED
+                --add-opens=java.base/java.time=ALL-UNNAMED
+                --add-opens=java.base/java.text=ALL-UNNAMED
+                --add-opens=java.management/sun.management=ALL-UNNAMED
+                --add-opens java.desktop/java.awt.font=ALL-UNNAMED
+              </argLine>
+              <systemPropertyVariables>
+                <angela.java.opts>
+                  --add-opens=java.base/jdk.internal.access=ALL-UNNAMED
+                  --add-opens=java.base/jdk.internal.misc=ALL-UNNAMED
+                  --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
+                  --add-opens=java.base/sun.util.calendar=ALL-UNNAMED
+                  --add-opens=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED
+                  --add-opens=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED
+                  --add-opens=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED
+                  --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED
+                  --add-opens=java.base/java.io=ALL-UNNAMED
+                  --add-opens=java.base/java.nio=ALL-UNNAMED
+                  --add-opens=java.base/java.net=ALL-UNNAMED
+                  --add-opens=java.base/java.util=ALL-UNNAMED
+                  --add-opens=java.base/java.util.concurrent=ALL-UNNAMED
+                  --add-opens=java.base/java.util.concurrent.locks=ALL-UNNAMED
+                  --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED
+                  --add-opens=java.base/java.lang=ALL-UNNAMED
+                  --add-opens=java.base/java.lang.invoke=ALL-UNNAMED
+                  --add-opens=java.base/java.math=ALL-UNNAMED
+                  --add-opens=java.sql/java.sql=ALL-UNNAMED
+                  --add-opens=java.base/java.lang.reflect=ALL-UNNAMED
+                  --add-opens=java.base/java.time=ALL-UNNAMED
+                  --add-opens=java.base/java.text=ALL-UNNAMED
+                  --add-opens=java.management/sun.management=ALL-UNNAMED
+                  --add-opens java.desktop/java.awt.font=ALL-UNNAMED
+                  -Dangela.agent.debug=false
+                  -Djdk.net.hosts.file=${project.build.directory}/fake-hosts.txt
+                  -DIGNITE_UPDATE_NOTIFIER=false
```

Comment:
It's the only way ? 😨 
It will be painful to setup angela now. 
I'm also thinking of all the modules that will need such adaptation in platform and tc db tests for example.

Reply:
Yes... it's a bit of a mess... but at least it's one time changes. I'm already in the process of fixing the TcDB stuff it's what led me here.